tools/batched_lastz/batched_lastz.xml

```diff
@@ -22,7 +22,12 @@
     <tests>
       <test expect_num_outputs="1">
         <param name="input" value="input.tgz" ftype="tgz"/>
-        <output name="output" decompress="true" file="output.maf.gz" ftype="maf"/>
+        <output name="output">
+            <assert_contents>
+                <has_size min="1"/>
```

Comment:
@richard-burhans can I help here to make those tests more stringent and not less :)

You can for example assert the content of an uncompressed maf.